### ISSUE TITLE
bazel test ... fails

### ISSUE THREAD

**Issue by blackliner** [protocolbuffers/protobuf#6997]
Isn't the repo supposed to work out of the box, i.e.
```
git clone git@github.com:protocolbuffers/protobuf.git
cd protobuf
bazel test ...
```
I get this error, pointing to some gmock issues?
```
ERROR: <somewhere>/protobuf/WORKSPACE:25:1: //external:gtest depends on @submodule_gmock//:gtest in repository @submodule_gmock which failed to fetch. no such package '@submodule_gmock//': No WORKSPACE file found in <somewhere>/172c83d5655e7b67c62e3888010a602f/external/submodule_gmock
ERROR: <somewhere>/protobuf/WORKSPACE:30:1: //external:gtest_main depends on @submodule_gmock//:gtest_main in repository @submodule_gmock which failed to fetch. no such package '@submodule_gmock//': No WORKSPACE file found in <somewhere>/172c83d5655e7b67c62e3888010a602f/external/submodule
```

**Comment by dlj-NaN** [comment]
FYI, this was addressed in #7237, but the default hasn't landed yet.

For now, you can verify with:

```shell
$ bazel test --@com_google_protobuf//:incompatible_use_com_google_googletest=true ...
```

The default is updated in #7907, tentatively targeted for 3.14.0 (or HEAD, if you use HEAD).

**Comment by elharo** [comment]
fixed
